src/_product-updates/2024-07-25-access-management.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Access management
+tags: [Improvement]
+image: /images/changelog/changelog-access-management.png
+---
+
+![changelog-entry-page.png](/images/changelog/changelog-access-management.png)
+
+It's been a while since we added the first pillar of our access management system. It gave admins a way to precisely manage their users' access.
+
+As we work with increasingly larger companies, the way we handled user administration needed to evolve.
+
+Say hello to the improved access management page, designed to easily manage members and guests from a single place. 
+The new table shows more users per page and is easier to scan than before. You can also see which resources guests have access to directly from this table and globally remove their access.
+
+On another note, we know that SSO can be very handy for companies with a lot of users. We now support SSO as a login method, a feature we developed by working closely with our clients. The SSO login page can now be customized with your company's logo, providing a seamless experience for your users. [Learn more about SSO here →](https://docs.bump.sh/help/organizations/organization-access-management/#sso)
```

Comment:
This sounds like we did not support SSO at all before. We should try to rephrase so it's clear that there can be two SSO connections now: one for internal users/members, and one for external users/guests.